test/pipelines/test_pipeline_yaml.py

```diff
@@ -141,6 +142,7 @@ def test_load_yaml(tmp_path):
     assert isinstance(pipeline.get_node("reader"), MockReader)
 
 
+@pytest.mark.unit
 def test_load_yaml_elasticsearch_not_responding(tmp_path):
     # Test if DocumentStoreError is raised if elasticsearch instance is not responding (due to wrong port)
```

Comment:
That's in the gray area for me as a unit test. Testing for ElasticsearchDocumentStore and failing because of the wrong port. It's fine to keep it as is. Usually I would see this test together with a second test where the port is correct and it thus works, like a test_load_yaml_elasticsearch_responding.

Reply:
Alright, happy that it paid off to look through the tests one by one. 😄